### PR TITLE
ol - filter by skill

### PR DESCRIPTION
**Pull request by OrlandoLee**
@sradu wondering if you can take a look at this and I am wondering why the style is showing up for the button

inipRc3xbs

### CODE REVIEW

**Comment by OrlandoLee** [comment]
@sradu added project types tagging and filters as well.

just can not get the styling  right

**Comment by adrianthedev** [comment]
The new styling is up. @sradu please review

image

File: app/frontend/svgs/filter.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (sradu) [inline app/frontend/svgs/filter.svg]: For consistency's sake, TailwindUI uses the font pack below and I would copy and paste the raw values:

https://github.com/refactoringui/heroicons

I really did how it looks now, so I'm 100% for leaving this as is, but I'd be curios if that pack has something similar so we can be consistent.

Comment (adrianthedev) [reply app/frontend/svgs/filter.svg]: got it. I wasn't aware about heroicons. will refactor.

**Comment by adrianthedev** [comment]
done @sradu. FYI, you need to do `bundle install`

**Pull request merged**